stats.go

```diff
@@ -196,13 +196,26 @@ func (e *statsExporter) makeReq(vds []*view.Data, limit int) []*monitoringpb.Cre
 
 	for _, vd := range vds {
 		for _, row := range vd.Rows {
-			ts := &monitoringpb.TimeSeries{
-				Metric: &metricpb.Metric{
-					Type:   namespacedViewName(vd.View.Name),
-					Labels: newLabels(row.Tags, e.taskValue),
-				},
-				Resource: resource,
-				Points:   []*monitoringpb.Point{newPoint(vd.View, row, vd.Start, vd.End)},
+			var ts *monitoringpb.TimeSeries
+			switch vd.View.Aggregation.Type {
+			case view.AggTypeLastValue:
+				ts = &monitoringpb.TimeSeries{
+					Metric: &metricpb.Metric{
```

Comment:
Does this whole case need to be duplicated? Looks like the only difference is in the value of the Points field. If this is the case, could we instead just conditionally construct different Points values for the different aggregation types instead of the whole TimeSeries?

Reply:
I update this part in the most recent commit.

Reply:
If you think it's good, I would merge it. :)